CommunityToolkit.Common/Extensions/TaskExtensions.cs

```diff
@@ -2,6 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if NET6_0_OR_GREATER
+using System.Diagnostics.CodeAnalysis;
+#endif
 using System.Reflection;
 using System.Runtime.CompilerServices;
 using System.Threading.Tasks;
```

Comment:
Can we put all the conditional `using` below the top-level ones? If possible, with one line spacing!?

Reply:
We're doing this in all other files, so just keeping it simple for consistency here.

Reply:
Is the preferred style? If so, why? Almost all of the projects from .NET keep conditional `using` below the top-level ones.

Reply:
I'm not saying this is necessarily the preferred style, just the one we've used so far. This PR is just mean to add a missing annotation, so I wouldn't want to do additional unrelated refactoring here. We might want to change this in the future, but this PR is just not the right place to do that anyway, so that's why I'm leaving this bit as is for now 🙂